src/main/java/org/springframework/samples/petclinic/rest/VisitRestController.java

```diff
@@ -105,6 +106,8 @@
 		currentVisit.setDate(visit.getDate());
 		currentVisit.setDescription(visit.getDescription());
 		currentVisit.setPet(visit.getPet());
+        currentVisit.setAdHoc(visit.getAdHoc());
+        currentVisit.setScheduled(visit.getScheduled());
```

Comment:
Same as with create visit, might need to add other constraints check, so better to move into service.